vcfutils.c

```diff
@@ -256,10 +256,10 @@ int bcf_remove_allele_set(const bcf_hdr_t *header, bcf1_t *line, const struct kb
     int *map = (int*) calloc(line->n_allele, sizeof(int));
     uint8_t *dat = NULL;
 
+    bcf_unpack(line, BCF_UN_FMT);
```

Comment:
Should this be `BCF_UN_ALL`?  Looking at [`bcf_unpack()`](https://github.com/samtools/htslib/blob/f867feab/vcf.c#L3060), `BCF_UN_FMT` will only unpack the `indiv` data, but this needs to also access `info` and `allele`.

Reply:
Of course it should, duh

Reply:
This is now replaced with the correct `BCF_UN_ALL`